ext/ox/extconf.rb

```diff
@@ -19,15 +19,15 @@
   'RUBY_VERSION_MICRO' => version[2],
 }
 
-dflags.each do |k,v|
+dflags.each do |k, v|
```

Comment:
This is an odd format rule. Can the rule be made to keep k and v on the same line like other code changes?

Reply:
Do you mean, no space after a comma? I'm not seeing a multi-line change here

Reply:
That's odd. In the GitHub viewer the was on a separate line. It's fine the way it appears here.